distributed/distributed-schema.yaml

```
@@ -770,6 +770,12 @@ properties:
         - "null"
         description:
           The size of the memory pool in bytes
+      logging:
+        type:
+        - bool
```

I think the errors are related to the schema checking. 
```suggestion
        - boolean
```

---

Thanks! :)